autonomous/simple.py

```diff
@@ -0,0 +1,25 @@
+from magicbot import AutonomousStateMachine, timed_state
+from wpilib import RobotBase
+
+from components.chassis import ChassisComponent
+from controllers.note import NoteManager
+
+from utilities.game import is_red
+
+
+class SimpleAuto(AutonomousStateMachine):
+
+    chassis: ChassisComponent
+    note_manager: NoteManager
+
+    @timed_state(first=True, duration=5, next_state="drive_forward")
+    def shoot_note(self) -> None:
+        self.note_manager.try_shoot()
+
+        if self.note_manager.has_just_fired() or RobotBase.isSimulation():
```

Comment:
Since this is a `timed_state` anyway, we don't really need the `isSimulation` check.
```suggestion
        if self.note_manager.has_just_fired():
```